### PR TITLE
Fix #11 Update user guide for DSOL 2.0

### LINKED ISSUE #11: Update the TCK User Guide for Jakarta EE 9
Review and update User Guide to remove/replace obsolete references and update for inclusion with Jakarta EE 9

### PR DESCRIPTION
**Pull request by markt-asf**
I used the Jakarta Annotations API as a basis and generally aimed to update DSOL to align with it apart from where DSOL needed to diverge because it doesn't used the standard test harness.

### CODE REVIEW

File: userguide/src/main/jbake/assets/_config.yml [hunk 1]
```diff
@@ -1,14 +1,13 @@
-theme: jekyll-theme-architect
+remote_theme: jakartaee/jekyll-theme-jakarta-ee
 
-title: [tck_jaxrs_v2_1]
-description: [Oracle Technology Compatibility Kit User's Guide for Java API for RESTful Web Services (JAX-RS) 2.1 for Technology Licensees, Release 2.1]
+title: [tck_dsol_v2_0]
+description: [Jakarta EE Technology Compatibility Kit User's Guide for Jakarta Debugging Upport for Other Languages (DSOL), Release 2.0]
```
Comment (alwin-joseph) [inline userguide/src/main/jbake/assets/_config.yml]: typo "Jakarta Debugging **Upport**"

File: userguide/src/main/jbake/content/attributes.conf [hunk 2]
```diff
@@ -1,27 +1,33 @@
...
+// Java SE version required.
 :SEversion: 8
 :JavaTestVersion: T.T.T
 :jteFileName: <TS_HOME>/bin/wombat.jte
 :jtxFileName: <TS_HOME>/bin/wombat.jtx
-:TCKPackageName: wombat-x.x_dd-Mmm-YYYY.zip
-////
-Directory names used in examples in using.adoc.
-////
+:TCKPackageName: debugging-tck-2.0.0.zip
```
Comment (alwin-joseph) [inline userguide/src/main/jbake/content/attributes.conf]: I think the tck name expected here is the eftl versioned as "jakarta-debugging-tck-2.0.0.zip"

**Review by alwin-joseph — APPROVED** [review]
Thanks for helping with this. please address the comments too

**Comment by alwin-joseph** [comment]
Hi @markt-asf - Thanks for this change.
Could you please fix the minor issues as per the comments, so this can be merged. 

Hi @edbratt Please feel free to review

File: userguide/src/main/jbake/assets/_config.yml (hunk 1 above)
Comment (markt-asf) [reply userguide/src/main/jbake/assets/_config.yml]: Fixed.

File: userguide/src/main/jbake/content/attributes.conf (hunk 2 above)
Comment (markt-asf) [reply userguide/src/main/jbake/content/attributes.conf]: Fixed.

**Pull request merged**